logstash-core/lib/logstash/api/lib/app/service.rb

```diff
@@ -40,4 +44,15 @@ def get(key)
     end
     LogStash::Json.dump(data)
   end
+
+  private
+
+  def has_counters?
```

Comment:
Well, we can also change the behavior if the collector didn't collect any new metrics we dont send the snapshot?